js/index.js

```diff
@@ -68,21 +68,21 @@ messageForm.item(0).addEventListener("submit",(event) => {
       messageForm.item(0).reset();
 })
 
-var githubRequest = new XMLHttpRequest();
 
-githubRequest.open('GET', 'https://api.github.com/users/Manizha-khorram/repos');
-githubRequest.send();
-    githubRequest.onreadystatechange = function () {
-        if (githubRequest.readyState === 4) {
-            var repositories = JSON.parse(githubRequest.responseText);
-            console.log('i am repositories', repositories);
-        
-            let projectSection = document.getElementById('projects');
-            let projectList = projectSection.querySelector('ul');
-            for(let i = 0; i < repositories.length; i++){
-            let project = document.createElement('li');
-            project.innerText = repositories[i].name;
-            projectList.appendChild(project);
-   }        
-}}  
-      
+
+
+    
+fetch('https://api.github.com/users/Manizha-khorram/repos')
+ .then(response => response.json())
+ .then( data => {
+    
+     let projectSection = document.getElementById('projects');
+     let projectList = projectSection.querySelector('ul');
```

Comment:
I couldn't get the projects to display locally, I think it might be because the `#projects` section and `ul` are missing from the markup in `index.html`? :thinking: :slightly_frowning_face: